darts/timeseries.py

```diff
@@ -31,6 +32,9 @@ def __init__(self,
         ----------
         df
             The actual time series, as a pandas DataFrame with a proper time index.
+        columns
```

Comment:
Minor nitpick: I think the name `column_names` instead of `columns` might be a bit more descriptive of what this argument represents.

Reply:
while it is true that `column_names` is more descriptive I don't know if we want to deviate from pandas `columns` syntax ?

Reply:
Ah, good point!